scripts/install.sh

```diff
@@ -25,3 +25,9 @@ if [[ "${TOX_ENV}" == "pypy" ]]; then
     pyenv install pypy-2.6.0
     pyenv global pypy-2.6.0
 fi
+
+if [[ "${TOX_ENV}" == "gae" && ! -d ${GAE_PYTHONPATH} ]]; then
+    python scripts/fetch_gae_sdk.py `dirname ${GAE_PYTHONPATH}`
```

Comment:
RE: Stale version, I agree. This is the same locally. I'll need to update `fetch_gae_sdk` to do a better job, but we can do that in another PR.